index.html

```diff
@@ -8,7 +8,41 @@
     <title>Document</title>
   </head>
   <body>
-    <section class="advantages">
+    <header class="header container">
+      <nav class="nav__panel">
+        <div class="nav__panel--logo" src="./images/Triphouse_logo.png" alt="Triphouse logo"></div>
+        <div class="nav__panel--items">
+          <ul class="nav__panel--links ">
+            <li class="nav__panel--link _active">Stays</li>
+            <li class="nav__panel--link">Attractions</li>
+          </ul>
+          <div  class="nav__panel--buttons">
+            <div class="nav__panel--day-night-switcher"></div>
+            <div class="nav__panel--account-avatar"></div>
+          </div>
+        </div>
+      </nav>
+        <h1 class="header__title">Discover stays <br>to live, work or just relax</h1>
+        <form class="search__form" method="get">
+            <p class="search__form--destination">
+              <label class="search__form--title" for="destination">Your destination or hotel name</label>
+              <input required class="search__form--input-field" id="destination" type="text" name="destination" placeholder="Enter a destination or hotel name" value="New York" >
```

Comment:
All your inputs must have yellow border when active, not black. And grey if inactive. Accordinaglywith design

Reply:
> All your inputs must have yellow border when active, not black

Got it, this 👉 https://github.com/pavel-4/if-html/pull/5/commits/eb602cf846c6d24a335e1248737e0824edc43006 will solve the issue

> And grey if inactive. Accordinaglywith design

As for the grey borderd for input fields in its inactive state, I have doubts, therefore, with a strong increase in the layout, the _inactive field does not have grey borders_ 👇
<img width="300" alt="Снимок экрана 2022-07-23 в 14 19 31" src="https://user-images.githubusercontent.com/71784588/180603011-ec82d942-29fe-4893-a7bd-b72b04d07337.png">
Otherwise two inactive input fields, standing together will be looking like 👇
<img width="300" alt="image" src="https://user-images.githubusercontent.com/71784588/180603041-58822949-c8c4-47e7-887d-c663c3397681.png">

That was the reason why I'd added a pseudo-element to all input fields with grey borders, and it looks accordingly with design 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/71784588/180603121-055848c4-11cc-4830-8cb3-4612aab216fa.png">

 cc @almuly

Reply:
But if your field is active it is have black border.  on design yellow brder show you, that it is active, and if it is inactive, it should be grey

Reply:
> But if your field is active it is have black border.

That's strange to read, because I'd create a pseudo element`:focus` for active input fields (`styles.css`, lines 146 - 150) cc @almuly